package.json

```diff
@@ -10,22 +10,22 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": ">=16",
+    "node": ">=20",
```

Comment:
There is one other reference to Node.js v16 in 
https://github.com/bufbuild/buf-setup-action/blob/71c62afa859a83fef3e99267971d9b67716ff328/.nvmrc

It's used by https://github.com/nvm-sh/nvm and https://github.com/Schniz/fnm

We should bump this to `v20`.